nl/mail/chrome/messenger/filter.properties

```diff
@@ -53,19 +53,19 @@ filterAction17=gelabeld
 #entities below were automatically added by l10nmerge
 
 
-bodyFilterWarning=Creating a Body filter will automatically mark the Inbox for Offline use in Folder Properties. For this filter to operate on other folders, you must manually set those folders for Offline use in Folder Properties.
-bodyFilterWarningCheck=Don't show this warning again
-systemSound=System Sound
-soundFilePickerTitle=Choose Sound
-installScriptCheck=Yes, please install it
-noScriptSelected=Select an AppleScript
+bodyFilterWarning=Het aanmaken van een Body filter markeert automatisch de Inbox voor Offline gebruik in de Mapeigenschappen. Om dit filter toe te passen op andere mappen, moet u deze mappen handmatig op Offline gebruik zetten in de Mapeigenschappen.
+bodyFilterWarningCheck=Toon deze waarschuwing niet opnieuw
+systemSound=Systeemgeluid
+soundFilePickerTitle=Kies geluid
+installScriptCheck=Ja, installeer het
+noScriptSelected=Selecteer een AppleScript
 bodyFilters=Body Filters
-filterAction18=removed topic
-scriptFilePickerTitle=Choose Script
-filterAction20=mark as pending
-installScripLabel=You have selected %S. Do you want to install it in your personal Scripts folder?
-scriptFiles=AppleScript Files
-filterAction19=played sound
-installScript=Install Script in your personal Scripts folder?
-filterAction21=launched script
-filterAction22=mark as
+filterAction18=verwijderd onderwerp
+scriptFilePickerTitle=Kies Script
+filterAction20=markeer als hangend
```

Comment:
CORRECTION NEEDED. Depending on context it should be ''Markeer als in behandeling''.